src/test/java/com/google/javascript/clutz/crossModuleImports/goog_multi_provide_exporter.d.ts

```diff
@@ -0,0 +1,46 @@
+declare namespace ಠ_ಠ.clutz {
+  class module$exports$goog$multi$provide$exporter$FirstMultiProvideClass extends module$exports$goog$multi$provide$exporter$FirstMultiProvideClass_Instance {
+  }
+  class module$exports$goog$multi$provide$exporter$FirstMultiProvideClass_Instance {
+    private noStructuralTyping_: any;
+  }
+}
+declare namespace ಠ_ಠ.clutz.module$exports$goog.multi.provide.exporter {
+  export import FirstMultiProvideClass = ಠ_ಠ.clutz.module$exports$goog$multi$provide$exporter$FirstMultiProvideClass;
+}
+declare namespace ಠ_ಠ.clutz.module$exports$goog$multi.provide.exporter {
+  export import FirstMultiProvideClass = ಠ_ಠ.clutz.module$exports$goog$multi$provide$exporter$FirstMultiProvideClass;
+}
+declare namespace ಠ_ಠ.clutz.module$exports$goog$multi$provide.exporter {
+  export import FirstMultiProvideClass = ಠ_ಠ.clutz.module$exports$goog$multi$provide$exporter$FirstMultiProvideClass;
```

Comment:
This seems to be the same code repeated 3x

Reply:
If you look very carefully, some dollar signs turned into dots.  That's the secret for allowing goog.provide() style imports, where importing `foo` gives you access to `foo.bar.baz`, even with the goog.module()/dollar sign style symbol names.